extensions/resource-deployment/src/ui/deployClusterWizard/constants.ts

```diff
@@ -69,4 +69,7 @@ export const DockerRegistry_VariableName = 'AZDATA_NB_VAR_BDC_REGISTRY';
 export const DockerImageTag_VariableName = 'AZDATA_NB_VAR_BDC_DOCKER_IMAGE_TAG';
 export const DockerUsername_VariableName = 'AZDATA_NB_VAR_BDC_DOCKER_USERNAME';
 export const DockerPassword_VariableName = 'AZDATA_NB_VAR_BDC_DOCKER_PASSWORD';
+export const KubeCtlToolName = 'kubectl';
```

Comment:
these consts should be added to a different file, this file is for deploy bdc wizard but the tools are independent of it.